### DISCUSSION TITLE
Get live standings for a ROTO NBA league

### DISCUSSION THREAD

**Discussion by MihaMuskinja** [cwendt94/espn-api#439] [ESPN API]
I was working on my code for getting the standings in a roto league using the following call:

```
    league = api.get_league()
    params = {'view': ['mScoreboard']}
    data = league.espn_request.league_get(params=params)
```

However, this only shows the updated standings a few hours after all games have been finished. On the other hand, I am able to see the standings change in real-time on https://fantasy.espn.com/basketball/league/standings?leagueId=123456. Is there a way to access the live standings with the API?

**Comment by cwendt94** [comment]
After initiating the league you should be able to get standings using the `league.standings()` functionality. But like you said this is not live and will just have the data from when the league was initiated.

I could probably add a input to that function call for `live` flag to get the latest standings calling their API
